test/dynamo/test_dynamo.py

```diff
@@ -156,6 +156,22 @@ def forward(self, index, copy_tensor, input_tensor, op_name):
           xla_index, xla_copy_tensor, xla_input_tensor, op_name=in_place_op)
       self.assertTrue(torch.allclose(res_cpu, res_xla_dynamo.cpu()))
 
+  def test_einsum(self):
+    # einsum currently does not have meta function to compute the shape hence
+    # will fallback to XLA with FakeTensor as input to infer the output shape.
```

Comment:
fallback to CPU?

Reply:
it is actually fallback to XLA. so it is 
```
Meta tensor ---fallback--> XLA --fallback--> CPU
```